.github/workflows/regression_test_summariseEpoch.yml

```diff
@@ -0,0 +1,43 @@
+name: Python - summariseEpoch.py
+
+on: [push]
```

Comment:
Change this to
```python
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```
so that this workflow is also run when a PR is being requested.